lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,144 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid $gray-l3;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
```

Comment:
change to padding: ($baseline/4); and remove width line above this. This also allows you to remove the padding rule on .courseware-bookmarks-button